conbench/app/_util.py

```diff
@@ -70,6 +70,8 @@ def set_display_time(benchmark):
 
 def set_display_mean(benchmark):
     """
+    this needs to go away. :)
+
```

Comment:
This patch is a great reminder that we should do https://github.com/conbench/conbench/issues/1341.